src/datastore/team.hpp

```diff
@@ -30,27 +28,23 @@ class Team
 
     /* Constructors */
     Team();
-    Team(const std::string& name, League);
 
```

Comment:
Wait, why do you even want to do this, isn't it better to use a ctor then to have to go through extra layers of setter function calls when instantiating an object. Also, why the did you make league public, shouldn't it just be private and have getters and setters to be consistent with the other classes